playbooks/common/openshift-cluster/upgrades/v3_9/upgrade_control_plane.yml

```diff
@@ -76,6 +76,8 @@
       openshift_upgrade_target: '3.9'
       openshift_upgrade_min: '3.8'
       openshift_release: '3.9'
+      # Clearing openshift_version so the openshift_version role logic properly applies
+      openshift_version: ''
```

Comment:
We need to preserve the user's requested openshift_version, if applicable, similar to what we do with openshift_pkg_version.

Reply:
Users don't request `openshift_version`, we set this through the openshift_version role.  Users request `openshift_release`, `openshift_image_tag`, `openshift_pkg_version`.  The openshift_version role assumes that `openshift_version` will be unset and behaves differently when we go through that role a second time for the 3.9 upgrade.